### ISSUE TITLE
Sass deprecation warnings for use of RGB color functions

### ISSUE THREAD

**Issue by johncrim** [oddbird/accoutrement#133]
During our build with accoutrement 4.0.4 and sass 1.79.4 I'm seeing a number of deprecation warnings from code in the Accoutrement lib, like these:

```
Deprecation Warning: color.red() is deprecated. Suggestion:                                                                                                                                                                                                            

color.channel($color, "red", $space: rgb)

More info: https://sass-lang.com/d/color-functions

    ╷
225 │     'r': math.div(color.red($color), 255),
    │                   ^^^^^^^^^^^^^^^^^
    ╵
    node_modules\accoutrement\sass\color\_contrast.scss 225:19  luminance()
    node_modules\accoutrement\sass\color\_contrast.scss 335:15  best-contrast()
    node_modules\accoutrement\sass\color\_contrast.scss 84:11   contrast()
    theme\_color.scss 89:15                                     foreground()
    theme\_color.scss 167:12                                    Background()
    libs\style\test\theme-color.spec.scss 48:9                  @content


Deprecation Warning: color.green() is deprecated. Suggestion:

color.channel($color, "green", $space: rgb)

More info: https://sass-lang.com/d/color-functions

    ╷
226 │     'g': math.div(color.green($color), 255),
    │                   ^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules\accoutrement\sass\color\_contrast.scss 226:19  luminance()
    node_modules\accoutrement\sass\color\_contrast.scss 335:15  best-contrast()
    node_modules\accoutrement\sass\color\_contrast.scss 84:11   contrast()
    theme\_color.scss 89:15                                     foreground()
    theme\_color.scss 167:12                                    Background()
    libs\style\test\theme-color.spec.scss 48:9                  @content
```

I've already fixed similar warnings in our code, it's a pretty straightforward fix. I may be able to make the fix in this project, I just don't have time for it today.

**Comment by jgerigmeyer** [comment]
@johncrim Thanks for reporting! This should be fixed by #131, once we get that wrapped up.

**Comment by johncrim** [comment]
Hi - confirming that this and related issues are fixed in 4.0.5. Thank you!
